Add categories to Release Drafter

hugovk opened the pull request:
Follow on from #5058.

Changes proposed in this pull request:

 * Looking at the PRs listed at the [8.1.0 draft](https://github.com/python-pillow/Pillow/releases), they mainly fall into three categories:
   * Dependencies
   * Documentation
   * Testing

 * Let's configure Release Drafter to group PRs with these labels into categories with these titles

 * "Deprecations" and "Removals" would be useful for the future, to make those changes clear

 * Everything else falls into the blanket "Changes" section

 * By design, if a PR falls into multiple categories, it'll be listed in both. If we want to avoid it, we can use explicit `changelog: foo` labels

 * It is possible to have multiple labels per category too

radarhere commented on .github/release-drafter.yml:
```diff
@@ -2,6 +2,18 @@ name-template: "$NEXT_MINOR_VERSION"
 
+categories:
+  - title: "Dependencies"
+    label: "Dependencies"
```
```suggestion
    label: "Dependency"
```
Yes?

hugovk replied:
Yes, thanks!

https://github.com/python-pillow/Pillow/issues?q=label%3ADependency

hugovk commented:
The [8.1.0 draft](https://github.com/python-pillow/Pillow/releases) now looks like this (with `@`s removed to prevent spamming):

---

https://pillow.readthedocs.io/en/stable/releasenotes/8.1.0.html

## Changes

- Add Release Drafter to draft releases #5058 [hugovk]
- GHA: Install Raqm on macOS #5061 [nulano]
- GHA: Make windows cache depend on VS version #5057 [nulano]
- Moved warning to end of execution #4965 [radarhere]
- Add line breaks in test-windows #5052 [nulano]
- Removed unused C methods #5026 [radarhere]
- Remove excess Travis CI jobs #5029 [hugovk]
- init() if one of the formats is unrecognised #5037 [radarhere]
- Updated Python for diffcover and doccheck to 3.9 #5023 [radarhere]
- Updated macOS tested Pillow versions #5004 [radarhere]

## Dependencies

- Updated libraqm to 0.7.1 #5070 [radarhere]
- Updated libimagequant to 2.13.1 #5065 [radarhere]
- Update FriBiDi to 1.0.10 #5064 [nulano]
- Updated libraqm to 0.7.1 #5063 [radarhere]
- Updated libjpeg-turbo to 2.0.6 #5044 [radarhere]

## Documentation

- Add categories to Release Drafter #5081 [hugovk]
- Update CHANGES.rst #5078 [aclark4life]
- Corrected documentation syntax #5068 [radarhere]
- Updated macOS tested Pillow versions #5067 [radarhere]
- Updated instructions now that draft releases may exist #5066 [radarhere]
- Docs: add logo with text to menu #5055 [hugovk]
- Fixed link formatting #5040 [radarhere]
- PNG supports append_images #5031 [radarhere]
- Link directly to pilfont script #5030 [radarhere]
- Document FreeMono license #5002 [radarhere]
- Refer to "HPND License" instead of "PIL Software License" #5025 [radarhere]
- Added Fedora 33 to CI targets #5024 [radarhere]
- Update Travis CI links for .com #5022 [hugovk]
- Invoke pip using Python #5019 [radarhere]
- Updated CI targets #5018 [radarhere]
- Describe effect of WebP save_all #5016 [radarhere]
- Updated CI targets #5013 [radarhere]
- Clarified that "text" must be a string #5007 [radarhere]
- Releasing: Add a 'git push' to point release, adjust MD formatting #5006 [hugovk]

## Testing

- CI: Remove soon-EOL Amazon Linux 1 and CentOS 6 #5060 [hugovk]
- Updated extra test images list #5027 [radarhere]
- GHA: Add 3.10-dev #4980 [nulano]
- Add Fedora 33 #5021 [hugovk]
- Invoke pip using Python #5019 [radarhere]
- AppVeyor: Test on Python 3.9 final #5014 [hugovk]
- Travis CI: Test on Python 3.9 final #5010 [hugovk]
- CI: Replace test images from pillow-depends #5001 [nulano]
